### PR TITLE
Allow administrator to list namespaces and manage RBACs and admission webhooks

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

/area ops-productivity user-management
/kind enhancement
/priority normal
/invite @donistz @mvladev 

**What this PR does / why we need it**:
Allow landscape administrators to:
1. have all permission to read all namespaces
1. manage `ClusterRole(Binding)s`
1. manage `MutatingWebhookConfigurations` and `ValidatingWebhookConfigurations`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
`gardener.cloud:system:administrators` are now allowed to list namespaces, manage RBACs, admission webhooks and apiservices.
```

### CODE REVIEW

File: charts/gardener/controlplane/charts/application/templates/rbac-user.yaml [hunk 1]
```diff
@@ -22,6 +22,22 @@ rules:
   - namespaces
   verbs:
   - get
+  - list
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
+  - clusterrolebindings
```
Comment (rfranzke) [inline charts/gardener/controlplane/charts/application/templates/rbac-user.yaml:29]: What about `roles` and `rolebindings`?

Comment (vpnachev) [reply charts/gardener/controlplane/charts/application/templates/rbac-user.yaml:29]: They are implicitly granted by the clusterrole with label `gardener.cloud/role: project-member`.
I will add them explicitly.

**Review by donistz — DISMISSED** [review]

File: charts/gardener/controlplane/charts/application/templates/rbac-user.yaml [hunk 2]
```diff
@@ -22,6 +22,30 @@ rules:
   - namespaces
   verbs:
   - get
+  - list
+  - watch
```
Comment (timebertt) [inline charts/gardener/controlplane/charts/application/templates/rbac-user.yaml]: Should we allow all verbs on namespaces like for the other resources?

Comment (donistz) [reply charts/gardener/controlplane/charts/application/templates/rbac-user.yaml]: Developers/Admins don't have to be able to manage roles and bindings.

Comment (vpnachev) [reply charts/gardener/controlplane/charts/application/templates/rbac-user.yaml]: Done. With full permissions on RBACs it does not make sense to limit the administrator to only read the namespace resources when he/she can easily create new clusterrole(binding) to get these permissions.

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**